packages/connectivity/example/README.md

```diff
@@ -5,4 +5,4 @@ Demonstrates how to use the connectivity plugin.
 ## Getting Started
 
 For help getting started with Flutter, view our online
-[documentation](http://flutter.io/).
+[documentation](http://flutter.io/).
```

Comment:
As you haven't changed this file, it's better that you remove it from the PR.
(I think you removed the last line which was blank).

Reply:
You forgot this one, just append a blank line to this file and commit/push it.